servers/wasat/wg-keys.nix

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Just a thought: maybe we should make serokell/users and put both ssh and wg keys there, so that there's less stuff to change when HR want to offboard someone. Not necessarily now, but maybe a thing to discuss.

Reply:
To me having to push and redeploy infra repos on a serokell.nix update is annoying, I prefer keeping modules that are used in a single repo in that repo itself